code/modules/reagents/reagent_containers.dm

```diff
@@ -76,6 +87,34 @@
 	. = ..()
 	SplashReagents(hit_atom, TRUE)
 
+/obj/item/reagent_containers/proc/checkLiked(var/fraction, mob/M)
+	if(last_check_time + 50 < world.time)
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			var/total_foodtype = foodtype
+			if(reagents.reagent_list)
+				for(var/reagent in reagents.reagent_list)
+					var/datum/reagent/T = reagent
+					total_foodtype |= T.foodtype
+			if(!H.has_trait(TRAIT_AGEUSIA))

```

Comment:
no idea why this doesn't compile here

Reply:
has_trait was removed in favor of a macro, `HAS_TRAIT`

Reply:
so it has